src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -1115,6 +1110,27 @@ int Local2<U>()
                 // (67,32): warning CS8387: Type parameter 'T' has the same name as the type parameter from outer method 'C.M2<T>()'
                 //                     int Local3<T>() => 0;
                 Diagnostic(ErrorCode.WRN_TypeParameterSameAsOuterMethodTypeParameter, "T").WithArguments("T", "C.M2<T>()").WithLocation(67, 32));
```

Comment:
Isn't this a new diagnostic on existing code?

Reply:
That diagnostic was reported before. The only differences between C#7.3 (above) and C#8 (below) are the first two errors reported in C#7.3 but dropped in C#8.

---
In reply to: [245371945](https://github.com/dotnet/roslyn/pull/32067#discussion_r245371945) [](ancestors = 245371945)